index.js

```diff
@@ -519,6 +524,12 @@ function toggleSidebar() {
     allAboard.state('window', {
         badge: null
```

Comment:
You can do the below here i.e.
```
badge: null,
icon: {
    ...
}
```
It is also the documented way to do this https://developer.mozilla.org/en-US/Add-ons/SDK/Low-Level_APIs/ui_button_action#state()